pyk/src/pyk/prelude.py

```diff
@@ -24,6 +25,36 @@ def __init__(self):
         raise ValueError('Class Labels should not be instantiated')
 
 
+@final
+class Bool:
+
+    true: Final = TRUE
+    false: Final = FALSE
+
+    def __init__(self):
+        raise ValueError('Class Bool should not be instantiated')
+
+    @staticmethod
+    def of(b: bool) -> KToken:
+        return Bool.true if b else Bool.false
+
+    @staticmethod
+    def andBool(items: Iterable[KInner]) -> KInner:
+        return build_assoc(Bool.true, KLabel('_andBool_'), unique(items))
+
+    @staticmethod
+    def orBool(items: Iterable[KInner]) -> KInner:
+        return build_assoc(Bool.false, KLabel('_orBool_'), unique(items))
+
+    @staticmethod
+    def notBool(item: KInner) -> KApply:
+        return KApply(KLabel('notBool_'), [item])
+
+    @staticmethod
+    def impliesBool(antecedent: KInner, consequent: KInner) -> KApply:
+        return KApply(KLabel('_impliesBool_'), [antecedent, consequent])
```

Comment:
How should we stylize these? Should we just do `Bool.and`? Will that work in python or be confused with `and` operator.

Reply:
Unfortunately, `Bool.and` wouldn't work. I think this current naming is fine.

Reply:
But I think we should consider implementing this as a module. That way, depending on context, we can both

```python3
from prelude import bool

bool.andBool(...)
```

and

```python3
from prelude.bool import andBool

andBool(...)
```

Reply:
By a module you mean a new file like `pyk/prelude/bool.py`?

Reply:
Yes. exactly.